upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -468,15 +468,14 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 
 	if b.cluster.Spec.Networking.Calico != nil {
 		key := "networking.projectcalico.org"
-		version := "2.4.1-kops.1"
 
 		{
-			location := key + "/pre-k8s-1.6.yaml"
 			id := "pre-k8s-1.6"
+			location := key + "/" + id + ".yaml"
 
 			addons.Spec.Addons = append(addons.Spec.Addons, &channelsapi.AddonSpec{
 				Name:              fi.String(key),
-				Version:           fi.String(version),
+				Version:           fi.String("2.4.1"),
```

Comment:
changing this from `2.4.1-kops.1`, was there a specific reason the version was different? can't find the code for that specific tag. if so, should it be kept and and are some changes to be done also for 2.6.2?

Reply:
I don't actually know why that version exists, I got rid of it in my branch but maybe @chrislovecnm or @justinsb can enlighten us.

Reply:
This should be the version of calico.  We have another PR going in, which will force a rebase.  @itajaja are you able to update this afternoon?